### ISSUE TITLE
New Issue Template Not Being Applied 100% Of The Time

### ISSUE THREAD

**Issue by kylethielk** [ZenHubIO/support#1326]
Hi There,

Just wanted to report a bug that started popping up. The new issue template is working only some of the time. I haven't done enough scenario testing to see how to make this 100% reproducible, but I would say 90% of the time the new issue template simply doesn't load.

Here is the exception reported in Chrome Dev Tools each time it fails to work:

<img width="1022" alt="screen shot 2018-01-13 at 8 20 44 pm" src="https://user-images.githubusercontent.com/3186142/34912180-be733864-f89f-11e7-8583-565fc8a09f84.png">

Let me know if you need anymore details.

**Comment by kendramoroz** [comment]
Hey @kylethielk thanks for reporting this! I haven't had luck bringing up the same error - would you mind sharing more details about what's in your `ISSUE_TEMPLATE.d` file via support@zenhub.com?  This will help us re-create the same scenario you're running into to try better replicate the error. 

To confirm, are you experiencing this via extension, when creating Issues from the Board?

**Comment by kylethielk** [comment]
Hi @kendramoroz sure thing. There is nothing sensitive in there so I'm happy to share here. Its very basic

```
## Description

## Requested/Reported By

- [ ] 
```

If it helps this is in our project root and the file is called `ISSUE_TEMPLATE.md`. This repository is used 100% for tracking issues so the only other file in the entire repository is a `README.md`

**Comment by kendramoroz** [comment]
Thanks for the details! When you encountered the error (or encounter it ongoing), is there anything in particular you're changing about the Issue details? Certain file types or code snippets? or mostly just basic details?

**Comment by kylethielk** [comment]
Hi @kendramoroz  so in general I'm just creating a new issue from the Boards page in the Modal window.

From my recollection the "Create Issue" button actually stopped working for a few weeks and instead of creating a Modal simply opened Github's new issue page. Now the Zenhub modal opens to create the issue and that is when the Template issues started arising.

**Comment by Startouf** [comment]
Hello,
I have had the same problem for quite some weeks now and I thought the feature was actually broken.
This morning I have just seen this issue and tried again the new issue button and it was working fine. Our issue template is a bit bigger than kylethiek

I am thinking that the new issue template might break after some usage of the board, ie. opening/closing issues on the board. I will report any further feedback I have

**Comment by kylethielk** [comment]
Just to confirm what @Startouf  mentioned, it does seem to happen now only after performing a few actions on the board. I haven't been able to reproduce this from a fresh load of the board.

**Comment by kendramoroz** [comment]
🕵️‍♀️ We have this one on our radar, but as you've all mentioned, since it's not consistent it's been a sneaky bug! We'll keep our eyes open but if you come across it, any specifics about what you were doing or if there was anything in your template or comments that could help identify a cause, please continue to share!

**Comment by kendramoroz** [comment]
Checking in on this one! Anyone still consistently having issues when creating new issues?

**Comment by kylethielk** [comment]
HI @kendramoroz  it does still happen to me, but its only a minor inconvenience.

**Comment by kylethielk** [comment]
Absolutely. I'll start trying to track my steps every time I work in a Zenhub board so that when this issue crops up again, hopefully we can come up with a set or reproducible steps.

**Comment by kendramoroz** [comment]
Closing as this is quite old now, but let me know if you're still encountering trouble on this one!
